src/plugins/irc-events/connection.js

```diff
@@ -0,0 +1,75 @@
+var _ = require("lodash");
+var identd = require("../../identd");
+var Msg = require("../../models/msg");
+
+module.exports = function(irc, network) {
+	var client = this;
+
+	client.emit("msg", {
+		chan: network.channels[0].id,
+		msg: new Msg({
+			text: "Network created, connecting to " + network.host + ":" + network.port + "..."
+		})
+	});
+
+	irc.on("raw socket connected", function() {
+		identd.hook(irc.connection.socket, network.username);
```

Comment:
Associated with this change, could you remove [this line in `identd.hook()`](https://github.com/thelounge/lounge/blob/e0031dd67ec024753eebe473791532a1e81b8da0/src/identd.js#L12) so that the first argument is `socket` itself?
This is the only place where `identd.hook()` gets used.